environment_unix.yml

```diff
@@ -31,7 +31,7 @@ dependencies:
   - prometheus-cpp
   - libprotobuf < 4
   - openssl
-  - libcurl
+  - libcurl==8.5.0
```

Comment:
As discussed I don't think we should pin this

Reply:
After further discussion the pin will stay